root/types/metabrainz.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * @flow strict
+ * Copyright (C) 2017 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+/*
+ * Types are in alphabetical order.
+ *
+ * The definitions in this file are intended to model the output of the
+ * TO_JSON methods under lib/MusicBrainz/Server/Entity/, those are precisely
+ * how data is serialized for us.
+ */
+
+declare type BlogEntryT = {
+  +title: string,
+  +url: string,
+};
+
+declare type CritiqueBrainzReviewT = {
+  +author: CritiqueBrainzUserT,
+  +body: string,
+  +created: string,
+  +id: string,
+};
+
+declare type CritiqueBrainzUserT = {
+  +id: string,
+  +name: string,
+};
```

Comment:
I'd move these to root/static/scripts/common/components/CritiqueBrainzReview.js (exported). Only root/release_group/ReleaseGroupIndex.js needs to import CritiqueBrainzReviewT.